### PR TITLE
Client restructure angular

### PR DESCRIPTION
**Pull request by stephen-lt-sl**
@stevenhillcox @sievins @jebbinSCL

Restructures the client code to fit with the angular style guidelines.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,43 +3,46 @@
     <head>
         <meta charset="utf-8">
         <title>Chat</title>
-        <link href="main.css" rel="stylesheet" type="text/css">
+        <link href="app.css" rel="stylesheet" type="text/css">
         <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.1/angular.min.js"></script>
-        <script src="js/main.js"></script>
+        <script type="text/javascript" src="js/app.js"></script>
+        <script type="text/javascript" src="js/controllers/chat.controller.js"></script>
```
Comment (jebbinSCL) [inline public/index.html]: This is fine for now, but remember that you would want to use a bundler if this was a deployed application. It's up to you whether you use one here, but it is a good idea to figure out how they work with switching between a dev and production setup.

**Comment by jebbinSCL** [comment]
See comment about potentially using a bundler.
Add it to your list of things todo either way, but make a decision on whether to use one.

**Comment by stevenhillcox-sl** [comment]
Webpack bundling looks ok. Just need to resolve the merge conflicts.

**Comment by stephen-lt-sl** [comment]
Conflicts are resolved via tactical rebase (necessary because of a single comma), is it good to merge?

**Comment by jebbinSCL** [comment]
:ship:

**Pull request merged**